R/functions-imputation.R

```diff
@@ -70,11 +75,27 @@ imputeRowMin <- function(x, min_fraction = 1/2) {
 #' ignoring the intercept of the linear model estimating the relationship
 #' between standard deviation and mean. If `abs = TRUE` `NA` values are
 #' replaced with the absolute value of the random values.
+#'   
+#' For method **from_to**, imputed values are taken between 2 user defined 
+#' fractions of the row minimum.  
 #'
 #' @inheritParams imputeRowMin
+#' 
+#' @param method Imputation method.
```

Comment:
Please change this to: 

 method \`character(1)\` defining the imputation method. See description for details. Defaults to \`method = "mean_sd"\`.

It's always better to tell users what type of input is expected. Here for example we want the user to submit a text/character of length 1 defining the method.